<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Labor</class>
 <widget class="QWidget" name="Labor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>843</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>26</x>
     <y>29</y>
     <width>721</width>
     <height>381</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="Leaders">
    <attribute name="title">
     <string>决策人员</string>
    </attribute>
    <widget class="QTableView" name="tableView_leader">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>711</width>
       <height>361</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="SalesMen">
    <attribute name="title">
     <string>销售人员</string>
    </attribute>
    <widget class="QTableView" name="tableView_sale">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>711</width>
       <height>351</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>300</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="reserach_man">
    <attribute name="title">
     <string>研发人员</string>
    </attribute>
    <widget class="QTableView" name="tableView_reserch">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>821</width>
       <height>361</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="refresh">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>170</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>查看全表</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_person">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>50</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>增加人员</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_person">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>90</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>删除人员</string>
   </property>
  </widget>
  <widget class="QPushButton" name="save">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>130</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>保存修改</string>
   </property>
  </widget>
  <widget class="QPushButton" name="search">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>210</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>查找</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_search">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>240</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="return_2">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>270</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
